src/components/TransportCard.vue

```diff
@@ -40,12 +22,12 @@
 </template>
 
 <script>
-import VueEllipseProgress from "vue-ellipse-progress";
+import RadialProgressBar from "../../node_modules//vue-radial-progress";
```

Comment:
Quand tu va chercher une librairie qui se trouve dans node_modules le path n'est pas relatif
```suggestion
import RadialProgressBar from "vue-radial-progress";
```